packages/namekit-react/src/components/DomainCardPrice.tsx

```diff
@@ -0,0 +1,41 @@
+import { DisplayedPrice, DisplayedPriceProps } from "./DisplayedPrice";
+import {
+  CurrencySymbology,
+  getCurrencySymbology,
+  Price,
+} from "@namehash/ens-utils";
+import { Tooltip } from "./Tooltip";
+import React from "react";
+
+interface DomainCardPriceProps extends DisplayedPriceProps {
+  altPrice: Price;
+  altPriceSymbology: CurrencySymbology;
+}
+
+export const DomainCardPrice = ({
+  altPriceSymbology = CurrencySymbology.Symbol,
+  altPrice,
+  ...props
+}: DomainCardPriceProps) => {
+  return (
+    <Tooltip
+      trigger={
+        <DisplayedPrice
+          price={props.price}
+          symbol={props.symbol}
+          displaySize={props.displaySize}
+          symbolPosition={props.symbolPosition}
+        />
+      }
+    >
+      <DisplayedPrice
+        price={altPrice}
+        symbol={
+          <span>
+            {getCurrencySymbology(altPrice.currency, altPriceSymbology)}
```

Comment:
Suggest we update this to also use `CurrencySymbol` based on other suggestions I've shared. We can just pass in a custom `symbology` to that component if it is needed.

Reply:
perfect!